sprites/tagalong.asm

```diff
@@ -0,0 +1,1614 @@
+; ==============================================================================
+; Tagalong Library 
+; 
+; $1A00 - Tagalong YL
+; $1A14 - Tagalong YH
+; $1A28 - Tagalong XL
+; $1A3C - Tagalong XH
+; $1A50 - Tagalong Altitude
+; $1A64 - Tagalong Priority Layer
+
+
+; ==============================================================================
+    
+; $049E90-$049EF7 LONG JUMP LOCATION
+Tagalong_CheckBlindTriggerRegion:
+{
+    PHB : PHK : PLB
+    
+    LDX $02CF
+    
+    LDA $1A00, X : STA $00
+    LDA $1A14, X : STA $01
+    
+    LDA $1A28, X : STA $02
+    LDA $1A3C, X : STA $03
+    
+    STZ $0B
+    
+    LDA $1A50, X : STA $0A : BPL .non_negative_altitude
+      LDA.b #$FF : STA $0B
+  .non_negative_altitude
+    
+    REP #$20
+    
+    LDA $00 : CLC : ADC $0A : CLC : ADC.w #$000C : STA $00
+    LDA $02 : CLC : ADC.w #$0008 : STA $02
+    
+    LDA.w #$1568 : SEC : SBC $00 : BPL .non_negative_delta_x
+      EOR.w #$FFFF : INC A
+  .non_negative_delta_x
```

Comment:
labels should be on the same indentation of their opening "if" statement or loop
pretending they are the {} of a regular if
ex:
```CSharp
if ()
{
    code ...
}
```

```
LDA.b $00 : CMP.b #$01 : BNE .not1
    code ...

.not1

Reply:
Done